README.md

```diff
@@ -80,8 +80,12 @@ target is a url (http load tests) or host:port (grpc health test).  flags are:
   -grpc-port string
 	grpc server port. Can be in the form of host:port, ip:port or port.
 	(default "8079")
-  -grpc-secure
-	Use secure transport (tls) for GRPC
+  -cert
+	Full path to the server certificate required for secure grpc.
+	Used by grpcping and server. (default "") means secure grpc is disabled.
```

Comment:
add:

"unless https:// destination is used, in which case standard TLS is expected (with valid server certificates signed by the internet standard CAs)" 

or some such